mllib/src/main/scala/org/apache/spark/ml/feature/CountVectorizer.scala

```diff
@@ -100,6 +103,24 @@ private[feature] trait CountVectorizerParams extends Params with HasInputCol wit
 
   /** @group getParam */
   def getMinTF: Double = $(minTF)
+
+  /**
+    * Binary toggle to control the output vector values.
+    * If True, all nonzero counts (after minTF filter applied) are set to 1. This is useful for
+    * discrete probabilistic models that model binary events rather than integer counts.
```

Comment:
IMO, The sentence "This is useful...." can be removed. (sorry to add it in the first place).